pkg/yurtmanager/controller/yurtappoverrider/yurtappoverrider_controller.go

```diff
@@ -128,8 +153,35 @@ func (r *ReconcileYurtAppOverrider) Reconcile(_ context.Context, request reconci
 		return reconcile.Result{}, nil
 	}
 
-	deployments := v1.DeploymentList{}
+	if cacheOverrider, ok := r.CacheOverriderMap[instance.Name]; ok {
+		if reflect.DeepEqual(cacheOverrider.Entries, instance.Entries) {
+			return reconcile.Result{}, nil
+		}
+	} else {
+		r.CacheOverriderMap[instance.Name] = instance.DeepCopy()
+	}
```

Comment:
please move line156~162 after line 183, because we don't need to cache unsupported yurtappoverrier.

Reply:
fix